doc/ai-subnet.md

```diff
@@ -48,19 +48,22 @@ Orchestrators on the AI subnet can chose which models to support on their machin
     pip install huggingface_hub[cli,hf_transfer]
     ```
 
-3. **Create a Hugging Face Access Token**: Create a Hugging Face access token by fallowing the instructions in the [Hugging Face documentation](https://huggingface.co/docs/hub/en/security-tokens) and make this token available under the `HG_TOKEN` environment variable. This token will be used to download [token-gated models](https://huggingface.co/docs/transformers.js/en/guides/private) from the Hugging Face model hub.
+3. **Create a Hugging Face Access Token**: Create a Hugging Face access token by fallowing the instructions in the [Hugging Face documentation](https://huggingface.co/docs/hub/en/security-tokens) and make this token available under the `HF_TOKEN` environment variable. This token will be used to download [token-gated models](https://huggingface.co/docs/transformers.js/en/guides/private) from the Hugging Face model hub. *Additionally you can install your Hugging Face access token on your local machine using the Hugging Face CLI*
+```
+huggingface-cli login
```

Comment:
@papabear99 thanks for the pull request. Sadly for the AI runner the HF_TOKEN has to be set that is why I kept this out 🤔. Will update the docs a bit 👍🏻 .